android/envoy/src/main/kotlin/org/greatfire/envoy/EnvoyConnectionTests.kt

```diff
@@ -398,6 +537,64 @@ class EnvoyConnectionTests {
         return res
     }
 
+    suspend fun testV2RaySrtp(test: EnvoyTest): Boolean {
+        var addr = test.startService()
+
+        if (addr == "") {
+            // The go code doesn't handle failures well, but an empty
+            // string here indicates failure
+            test.stopService()
+            return false
+        }
+
+        test.proxyUrl = "socks5://$addr"
+        Log.d(TAG, "Testing V2Ray SRTP ${test.proxyUrl}")
+
+        val res = testStandardProxy(Uri.parse(test.proxyUrl))
+        if (res == false) {
+            test.stopService()
+        }
+        return res
+    }
```

Comment:
is there any difference between these 2 methods?  also expanding the surrounding code it seems like maybe this isn't new but was duplicated?